SConstruct

```diff
@@ -178,8 +178,21 @@ opts.Add(
         "optimize", "Optimization level", "speed_trace", ("none", "custom", "debug", "speed", "speed_trace", "size")
     )
 )
-opts.Add(BoolVariable("debug_symbols", "Build with debugging symbols", False))
-opts.Add(BoolVariable("separate_debug_symbols", "Extract debugging symbols to a separate file", False))
+opts.Add(
+    EnumVariable(
+        "debug_symbols",
+        "Build with debugging symbols",
+        "no",
+        ("no", "yes", "embedded", "separate"),
```

Comment:
```suggestion
        ("embedded", "separate") + TRUE_STRINGS + FALSE_STRINGS,
```

Reply:
The `--help` output would become unreadable :/